### ISSUE TITLE
Achieve 100% code coverage

### ISSUE THREAD

**Issue by billdenney** [billdenney/pknca#332]
Find all lines of code that are not covered (https://app.codecov.io/github/billdenney/pknca/tree/main/R) and add coverage for them.

**Comment by DBartlettHP** [comment]
These lines seem to impossible to cover:

https://github.com/billdenney/pknca/blob/4de6d420657b2513595fce32afeb2c3dd6c511d2/R/aucint.R#L77-L78

**Comment by billdenney** [comment]
Yes, that appears to be impossible to reach when clast is `NA`. Please change that code to a "Please report a bug" error.

**Comment by DBartlettHP** [comment]
These lines seem to also be impossible to cover:

https://github.com/billdenney/pknca/blob/b42d114c66207591dbd915e714347f106bfc291f/R/class-PKNCAdata.R#L132-L139

Notes:
- `ncol(current_group)` should always be greater than zero.
- `current_group` is the joined conc and dose data.

**Comment by DBartlettHP** [comment]
I am having a difficult time achieving coverage for:

https://github.com/billdenney/pknca/blob/cdc4c07e38e02154b9bc7f636c5bf81ffb170b89/R/class-summary_PKNCAresults.R#L288-L290

It may not be a nocov, but getting to this coverage is quite challenging.

Notes:
- `dplyr::summarize(
        N = length(unique(.data[[subject_col]]))
      )`
      
- N is derived within `get_summary_PKNCAresults_count_N`

- `get_summary_PKNCAresults_count_N` is used within `summary.PKNCAresults`

-  This eventually gets to `pk.nca` through many other functions.

**Comment by DBartlettHP** [comment]
I am not sure if the nocov here is working as intended. [Does not achieve testing coverage on lines 31 and 35 specifically]

https://github.com/billdenney/pknca/blob/cdc4c07e38e02154b9bc7f636c5bf81ffb170b89/R/dplyr.R#L31-L35

Should this be a nocov start and end?

**Comment by DBartlettHP** [comment]
> I am not sure if the nocov here is working as intended.

Same idea here:
https://github.com/billdenney/pknca/blob/cdc4c07e38e02154b9bc7f636c5bf81ffb170b89/R/general.functions.R#L193-L203

**Comment by billdenney** [comment]
This issue is sufficiently resolved at the moment by getting much closer to 100%.
